src/Tizen.Applications.Common/Tizen.Applications/LocaleManager.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Tizen.Applications
+{
+    static class LocaleManager
```

Comment:
This should be 'public'.

Reply:
Done :-) [commit](https://github.com/Samsung/TizenFX/pull/7174/commits/7d31ff5e35169be7bb972029d435226f3a0f1c33)